README.adoc

```diff
@@ -264,23 +250,23 @@ and artifacts that changed, and the server picks these up automatically without
 
 While the server is running in the foreground, it displays various console messages in the shell.
 These messages are also logged to the `target/liberty/wlp/usr/servers/defaultServer/logs/console.log`
-file. You can find the complete server logs under the `target/liberty/wlp/usr/servers/defaultServer/logs`
-directory. The primary log files in that directory are `console.log` and `messages.log` which contain
-console output of the running application and server. Additional logs are created when more serious
-errors occur or whenever tracing is enabled. These can be found under the `ffdc` directory and in the
-`trace.log` file respectively.
+file. You can find the complete server logs in the `target/liberty/wlp/usr/servers/defaultServer/logs`
+directory. The `console.log` and `messages.log` files are the primary log files in this directory and contain
```

Comment:
We can change it to..."The `console.log` and `messages.log` files are the primary log files that contain console output of the running application and the server."

The word "files" is a plural subject, so we need to use the verb to match, which is "contain" instead of "contains."

Reply:
ok, that sounds good